docs/api.md

```diff
@@ -1701,6 +1734,34 @@ puppeteer.launch().then(async browser => {
 });
 ```
 
+#### frame.waitForXpath(xpath[, options])
+- `xpath` <[string]> A [xpath] of an element to wait for
```

Comment:
Nit: I think xpath is capitalized as XPath for the link, and that would make it `waitForXPath` as the method name.

Reply:
done.